src/modules/10-common/pages/SettingsPages/OrgHeader.tsx

```diff
@@ -0,0 +1,234 @@
+      history.push(routes.toOrgs({ accountId }))
+    } catch (err) {
+      showError(getRBACErrorMessage(err))
+    } finally {
+      closeDialog()
+    }
+  }
+
+  const { openDialog, closeDialog } = useConfirmationDialog({
+    contentText: (
+      <String
+        // eslint-disable-next-line strings-restrict-modules
+        stringID="projectsOrgs.confirmDelete"
+        vars={{
+          name: organization?.name
+        }}
+        useRichText={true}
+      />
+    ),
+    // eslint-disable-next-line strings-restrict-modules
+    titleText: getString('projectsOrgs.confirmDeleteTitle'),
+    intent: Intent.DANGER,
+    customButtons: (
+      <DeleteProjectOrgButtons
+        onCancel={() => {
+          closeDialog()
+        }}
+        name={organization?.name || ''}
+        onDelete={onDeleteAction}
+        disableDeleteBtn={loading}
+        inputLabel={getString('projectsOrgs.toConfirmOrg')}
+        inputPlaceholder={getString('projectsOrgs.toDelete', { name: getString('orgLabel') })}
+        confirmBtnLabel={getString('projectsOrgs.confirmDeleteProjectOrg', {
+          type: getString('orgLabel').toLowerCase()
+        })}
+      />
+    ),
+    onCloseDialog: async (isConfirmed: boolean) => {
+      /* istanbul ignore else */ if (isConfirmed) {
+        onDeleteAction()
+      }
+    }
+  })
+
+  return (
+    <>
+      <Page.Header
+        className={css.pageHeader}
+        size="xlarge"
+        toolbar={
+          <Layout.Vertical padding={'large'} flex={{ justifyContent: 'space-between', alignItems: 'flex-end' }}>
+            <Popover
+              isOpen={menuOpen}
+              onInteraction={nextOpenState => {
+                setMenuOpen(nextOpenState)
+              }}
+              className={Classes.DARK}
+              position={Position.BOTTOM_RIGHT}
+            >
+              <Button
+                minimal
+                icon="Options"
+                onClick={e => {
+                  e.stopPropagation()
+                  setMenuOpen(true)
+                }}
+              />
+              <OrgPopoverMenu
+                org={organization as Organization}
+                setMenuOpen={setMenuOpen}
+                inviteCollab={() => openCollaboratorModal(identifier)}
+                editOrg={() => openOrganizationModal(organization as Organization)}
+                openDialog={openDialog}
+              />
+            </Popover>
+            <Layout.Horizontal padding={{ top: 'xlarge' }}>
+              <Layout.Vertical spacing="xsmall" flex>
+                <Text font="xsmall" padding={{ bottom: 'xsmall' }}>
+                  {getString('adminLabel')}
+                </Text>
+                <RbacAvatarGroup
+                  avatars={data?.data?.admins?.length ? data.data.admins : [{}]}
+                  onAdd={event => {
+                    event.stopPropagation()
+                    openCollaboratorModal(identifier)
```

Comment:
This seems to be incorrectly passed, with this syntax it does not have orgId and would rather treat this as an account invite modal. (please check all similar occurrence)

```suggestion
                    openCollaboratorModal({ orgIdentifier: identifier })
```